packages/reactotron-server/src/server/index.test.ts

```diff
@@ -0,0 +1,36 @@
+import test from "ava"
+import * as request from "request-promise-native"
+import { httpServerInstance, apolloServerInstance } from './'
+
+// Tests in here are ran in serial due to the webserver binding to port 4000 (concurrency results in EADDRINUSE error)
+
+let app, httpServer;
+
+test.serial("Can create a http server to serve react app", async t => {
+
+  let appServer;
+
+  t.notThrows(() => {
+    appServer = httpServerInstance()
+  })
+
+  // Not sure if there is a way to nicely destructure here, this does for now
+  app = appServer.app;
```

Comment:
If you guys have a good refactor for this, let me know! Scope wise I was running into an issue trying to get the app and httpServer properties within notThrows, but passing them out the function.